salesforce-connector-demo/src_hd/salesforce/connector/demo/ListOpps/ListOpps.xhtml

```diff
@@ -0,0 +1,147 @@
+									headerText="#{ivy.cms.co('/Dialogs/listOpps/accName')}"
+									width="100px;" sortBy="#{opp.accName}"
+									filterBy="#{opp.accName}" filterMatchMode="contains">
+									<h:outputText value="#{opp.accName}" />
+								</p:column>
+								<p:column
+									headerText="#{ivy.cms.co('/Dialogs/listOpps/amount')}"
+									width="100px;" sortBy="#{opp.amount}"
+									filterBy="#{opp.amount}" filterMatchMode="contains">
+									<h:outputText value="#{opp.amount}" />
+								</p:column>
+								<p:column
+									headerText="#{ivy.cms.co('/Dialogs/listOpps/closeDate')}"
+									width="100px;" sortBy="#{opp.closeDate}"
+									filterBy="#{opp.closeDate}" filterMatchMode="contains">
+									<h:outputText value="#{opp.closeDate}">
+										<f:convertDateTime pattern="dd.MM.yyyy" />
+									</h:outputText>
+								</p:column>
+								<p:column
+									headerText="#{ivy.cms.co('/Dialogs/listOpps/stage')}"
+									width="100px;" sortBy="#{opp.stage}"
+									filterBy="#{opp.stage}" filterMatchMode="contains">
+									<h:outputText value="#{opp.stage}" />
+								</p:column>
+								<p:column id="actionColumn" toggleable="false" headerText="#{ivy.cms.co('/Dialogs/listOpps/action')}" width="60px;" exportable="false" selectRow="false">
+									<p:commandLink id="edit" actionListener="#{data.bean.updateOppotunity(opp.id)}" oncomplete="PF('opportunityUpdateDialog').show()" update="#{p:resolveFirstComponentWithId('opportunityUpdatelForm', view).clientId}" title="Edit">
+										<i class="fa fa-pencil margin4" />
+									</p:commandLink>
+									<p:commandLink  id="delete" actionListener="#{logic.delete(opp.id)}" update="form" styleClass="textDecorationNone" title="Delete">
+										<i class="fa fa-trash margin4" />
+										<p:confirm header="Confirmation" message="Do you want to delete this record?" icon="pi pi-info-circle"/>
+									</p:commandLink>
+									<!-- <p:commandLink  id="newActivity" update="form" styleClass="textDecorationNone" title="New Activity">
+										<i class="fa fa-add margin4" />
+									</p:commandLink> -->
+								</p:column>
+							</p:dataTable>
+						</div>
+					</div>
+				</div>
+				<br />
+				<div class="command-btns">
+					<p:commandButton id="proceed" actionListener="#{logic.close}"
+						value="Close" update="form" icon="si si-close" />
+				</div>
+			</h:form>
+			
+			<p:confirmDialog global="true" showEffect="fade" hideEffect="fade" responsive="true" width="350">
+	            <p:commandButton value="No" type="button" styleClass="ui-confirmdialog-no ui-button-flat"/>
+	            <p:commandButton value="Yes" type="button" styleClass="ui-confirmdialog-yes" />
+	        </p:confirmDialog>
+			
+			<ui:include src="OppDetail.xhtml"></ui:include>
+			<ui:include src="UpdateOppotunity.xhtml"></ui:include>
+			<ui:include src="CreateOppotunity.xhtml"></ui:include>
+
+			<style>
+				.textDecorationNone {
+					text-decoration: none !important;
```

Comment:
ivy embedded the PrimeFlex-v2 https://github.com/primefaces/primeflex/wiki/PrimeFlex-v2
could you try to use it instead of introducing new CSS here, @HoDacVu90